subgraphs/truefi/src/mappings/ManagedPortfolioFactory.ts

```diff
@@ -0,0 +1,10 @@
+import { dataSource } from "@graphprotocol/graph-ts";
+import { PortfolioCreated } from "../../generated/ManagedPortfolioFactory/ManagedPortfolioFactory";
+import { ManagedPortfolio } from "../../generated/templates";
+
+export function handlePortfolioCreated(event: PortfolioCreated): void {
+  ManagedPortfolio.createWithContext(
```

Comment:
does `createWithContext()` differ from jsut creating a new template ? I am curious as I haven't used that function much

Reply:
If some parameters are needed to transfer, then createWithContext() is needed. Changed in d99eb453ec77afb674d3a370e1d0c781708ed260

Reply:
it doesn't look like there is any "context is actually set here though? 

https://thegraph.com/docs/en/developing/creating-a-subgraph/#data-source-context

Reply:
No, no context is set in the code. So I changed the function to create() in https://github.com/messari/subgraphs/commit/d99eb453ec77afb674d3a370e1d0c781708ed260